.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,19 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  golangci-pr:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
```

Comment:
`v1.38` (or `latest`, but particular version seems to be safer) - based on https://github.com/golangci/golangci-lint/blob/master/assets/github-action-config.json

Reply:
Good catch, I updated the version